cmd/root.go

```diff
@@ -16,33 +16,6 @@ import (
 // Version is the pgroll version
 var Version = "development"
 
-func init() {
```

Comment:
I checked and it's not mandatory to have the `init()` function accoring to some GitHub discussions in the spf13/cobra repo. Added a way that we can init the rootCmd without calling `execute` to introspect the commands.